R/pathogen_summary.R

```diff
@@ -14,14 +14,18 @@
 #' pathogen_summary(pathogen = "SARS_CoV")
 pathogen_summary <- function(pathogen) {
 
+  # read the data to get possible pathogen names
+  params <- utils::read.csv(system.file(
```

Comment:
Same as previously. Make sure you only read this once within this function.

Reply:
Resolved in 8427837b95fc718f7eed8f96283ee76bc925f917